### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by HanaSabih**
- Set up a new project with webpack
- Create an index.html file and write my HTML markup here
- Create an index.js file and set an array of some simple to-do tasks (array of objects). Each task object should contain three keys:

1. description [string].
2. completed [bool].
3. index: [number].

On page load render the dynamically created list of tasks in the dedicated placeholder
Create a style.css and set rules for the To-Do List
All your source files (index.html, index.js, and style.css)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import './style.css';
+// import bootstrap from "bootstrap";
```
Comment (rustam-tolipov) [inline src/index.js]: - [ OPTIONAL] Please ensure that any unused commented codes are removed from the file. Thank you.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+const getStorage = () => {
+  const storage = localStorage.getItem('ToDo') === null
+    ? []
+    : JSON.parse(localStorage.getItem('ToDo'));
+  return storage;
+};
+
+// array
+const arrToDos = getStorage();
```
Comment (rustam-tolipov) [inline src/index.js]: - [ ] So, At that point you have left one required change. 

> Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].

Kindly follow the insturction and create your dinamic array. Also, make sure to display them when the page loaded. Thank you.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @HanaSabih,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (HanaSabih) [reply src/index.js:21]: @Rustamxon7
yes i did it

Comment (HanaSabih) [reply src/index.js:21]: class ToDos {
  constructor(index = 0, description, completed = false) {
    this.index = index;
    this.description = description;
    this.completed = completed;
  }
}

Comment (HanaSabih) [reply src/index.js:21]: const ToDoItem = new ToDos(id, toDoInput.value, false);

Comment (HanaSabih) [reply src/index.js:21]: arrToDos.push(ToDoItem);

Comment (HanaSabih) [reply src/index.js:21]: @Rustamxon7
please can you check it

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @HanaSabih , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :heavy_check_mark: 
- [ ] Good UI design :heavy_check_mark: 
- [ ] No linter errors :heavy_check_mark: 
- [ ] Professional readme file :heavy_check_mark: 
- [ ] Descriptive pull request title and summary :heavy_check_mark: 
- [ ] Implemented changes from the previous reviewer :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**